### DISCUSSION TITLE
Is Hero9 still a valid device as a new buy?

### DISCUSSION THREAD

**Discussion by La-te** [gopro/labs#588] [Q&A]
There's a good deal on Hero9 at a local store. However, it seems to be dropped from GoPro Labs updates already. Is the latest available firmware for that one still a greatly working firmware? My main interest is in using it for Very Long Time-lapses with Larger Intervals https://gopro.github.io/labs/control/longtimelapse/.

**Comment by dnewman-gpsw** [comment]
Your assumption is correct, there are no more planned Labs updates for HERO9. The Very Long Time-lapses feature does work with this caveat: it is generally reliable when using an internal battery, get you 24+ hours of timelapse from a single battery charge (with a ~5 minute interval). The reliability issues are more under USB power, running these type of commands for weeks and months.

**Reply by La-te** [reply]
If running with power from the wall can I run with some other feature or a standard timelapse until the card is full with some 10-60 min interval?

**Reply by dnewman-gpsw** [reply]
Correct.  Having the camera turn on and off 1000+ times is the issue, recording timelapse continuously is not.  I have personally done that for 3 months on one project.
